docs/md/sessions.md

```diff
@@ -66,7 +66,12 @@ Is used when another session triggered this run using a trigger_flow action.
             "language": "",
             "timezone": "",
             "created_on": "0001-01-01T00:00:00Z",
-            "urns": []
+            "urns": [],
+            "fields": {
+                "gender": {
+                    "text": "Male"
+                }
+            }
```

Comment:
see below - this trigger previously had a non-existent field so it wasn't appearing when the trigger was later serialized